xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-notifications/xwiki-platform-legacy-notifications-notifiers/xwiki-platform-legacy-notifications-notifiers-api/src/test/java/org/xwiki/notifications/notifiers/internal/email/live/LiveNotificationEmailSenderTest.java

```diff
@@ -74,12 +74,12 @@ public void setUp() throws Exception
 
         this.mailListenerProvider = this.mocker.registerMockComponent(Provider.class, "database");
 
-        this.notificationUserIteratorProvider = mock(Provider.class);
+        this.notificationUserIteratorProvider = Mockito.mock(Provider.class);
```

Comment:
Not sure why you added Mockito. since mock call is already statically imported.

Reply:
Change provided by the IDE when I moved the class, I fixed it in some places but looks like I missed it here.